src/screens/add-education/common.ts

```diff
@@ -1,8 +1,9 @@
 const defaultAddEducationPayload = {
-  universityName: '',
-  courseName: '',
+  id: '',
+  specialization: '',
+  university: '',
   degree: '',
-  startDate: undefined,
```

Comment:
I'd leave it `undefined`

Reply:
`new Date().toISOString()` will allow user to skip additional date selection step in case if this education starts today. This is the same for the **experience**.
![image](https://user-images.githubusercontent.com/41474876/160849677-ee0a5600-7a26-4a6e-b655-6039944ab842.png)